scripts/run_msf_addons.sh

```diff
@@ -32,3 +32,6 @@ echo "$INFO Creating OpenFn admin user..."
 
 ) || echo "$ERROR Creating OpenFn admin user failed. Skipping"
 echo ""
+
+# this step requires internet. It downloads node modules. Should be run on the host
+# docker compose run worker npm install -g @openfn/cli && openfn repo install -a http@latest dhis2@latest openmrs@latest dhis2@5.0.1
```

Comment:
@midigofrank , my understanding is that this script is executed while they have internet—they're executing this remotely (from Switzerland, attached to a server in Iraq)

so... we should be good.